images/win/toolsets/toolset-2016.json

```diff
@@ -112,5 +112,34 @@
                 "4.3.0"
             ]
         }
-    ]
+    ],
+    "android": {
+        "platform_list": [
+            "android-30", "android-29", "android-28", "android-27", "android-25", "android-24", "android-23", "android-26", "android-22", "android-21", "android-19"
+        ],
+        "build_tools": [
+            "30.0.0", "29.0.3", "29.0.2", "29.0.1", "29.0.0", "28.0.3", "28.0.2", "28.0.1", "28.0.0", "27.0.3", "27.0.2", "27.0.1", "27.0.0", "26.0.3", "26.0.2", "26.0.1", "26.0.0", "25.0.3", "25.0.2", "25.0.1", "25.0.0", "24.0.3", "24.0.2", "24.0.1", "24.0.0", "23.0.3", "23.0.2", "23.0.1", "22.0.1", "21.1.2", "20.0.0", "19.1.0"
+        ],
+        "extra_list": [
+            "extras;android;m2repository",
+            "extras;google;m2repository",
+            "extras;google;google_play_services",
+            "extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.2",
+            "extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.1",
+            "extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2",
+            "extras;m2repository;com;android;support;constraint;constraint-layout;1.0.1"
+        ],
+        "addon_list": [
+            "add-ons;addon-google_apis-google-24",

```

Comment:
Why did we add the `add-ons` prefix here and skip it in the `toolset-2019.json` file?

Reply:
The same for the `extras` prefix

Reply:
Fixed, thank you!